web/concrete/blocks/page_title/form_add_edit.php

```diff
@@ -1,29 +1,140 @@
-<?php defined('C5_EXECUTE') or die("Access Denied.");
-if(!strlen($titleText)) { $titleText = $controller->getTitleText();}
-?>
-
-<div class="form-group" class="ccm-ui">
-    <label><?=t("Custom Title")?></label>
-    <div class="checkbox"><label>
-        <?php echo $form->checkbox('useCustomTitle', 1, $useCustomTitle);?>
-        <?=t('Override page name with custom title?')?>
-    </label></div>
+<?php defined('C5_EXECUTE') or die('Access Denied.'); ?>
+
+<div class="form-group">
+    <label><?php echo t("Custom Title"); ?></label>
+    <div class="checkbox">
+        <label>
+        <?php echo $form->checkbox('useCustomTitle', 1, $useCustomTitle); ?>
+        <?php echo t('Override page name with custom title'); ?>
```

Comment:
Changing translatable texts require that translators re-translate this string... Since it's an almost unnoticeable change, what about keeping the previous text?